packages/database/src/migrations/20220622064208-AddArrayConcatAggregationFunction-modifies-schema.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+exports.up = function (db) {
+  return db.runSql(`
+  CREATE OR REPLACE AGGREGATE array_concat_agg(anyarray) (
+    SFUNC = array_cat,
+    STYPE = anyarray
+  );
+  `);
```

Comment:
@edmofro there's actually a little start up problem when trying to deploy this migration.

In order to run this migration, the `MeditrakSyncQueue` must be running. 
In order to create the `MeditrakSyncQueue` the database must have the `array_concat_agg` function.

I guess this highlights a genuine dependency cycle in our code logic, however I feel it'll be acceptable for us to just manually install this function in the prod database prior to releasing this code?

Reply:
Yep I'm fine with that